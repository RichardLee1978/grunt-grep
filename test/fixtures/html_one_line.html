<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
</head>
<body>
	<div id='home'> <!--dev-->
	<div id='away'> <!--prod-->
	<span></span>
	<img src="./asd" alt=""> <!-- some string -->
</body>
</html>