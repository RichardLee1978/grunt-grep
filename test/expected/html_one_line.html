<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Document</title>
</head>
<body>
	<div id='home'> <!--dev-->
	<span></span>
	<img src="./asd" alt=""> <!-- some string -->
</body>
</html>